[SDK-2161] Allow to skip the redirect callback (#86)
auth0/auth0-angular: README.md

}
```

> ### Dynamic Configuration

Author: It looks like this header was wrong, updated it to be the same level of heading as the other titles.

],
```

### Using multiple OAuth providers

> If your application is making use of multiple OAuth providers, you might need to use multiple callback paths as well, one for each OAuth provider.

Reviewer: Edits for conciseness:

```suggestion
If your application uses multiple OAuth providers, you may need to use multiple callback paths as well, one for each OAuth provider.
```

> To ensure the SDK does not process the callback for any provider other than Auth0, you should configure the AuthModule by setting `skipRedirectCallback`.

Reviewer: ```suggestion
To ensure the SDK does not process the callback for any provider other than Auth0, configure the AuthModule by setting the `skipRedirectCallback` property as follows:
```
